guides/doc-Configuring_virt-who_VM_Subscriptions/master.adoc

```diff
@@ -0,0 +1,95 @@
+You cannot use the virt-who plugin without the Katello plugin.
+endif::[]
+
+ifdef::satellite[]
+include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
+endif::[]
+
+ifdef::katello,satellite,orcharhino[]
+include::common/assembly_overview-of-vm-subscriptions.adoc[leveloffset=+1]
+
+// To do
+// include::common/modules/ref_virt-who-configuration-examples.adoc[leveloffset=+2]
+
+:!virt-who-vm-subs:
+:context: kvm
+:kvm:
+include::common/modules/con_configuring-virt-who-for-provider.adoc[leveloffset=+1]
+
+include::common/modules/proc_creating-a-virt-who-configuration.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-hypervisor-smartproxy.adoc[leveloffset=+2]
+:!kvm:
+:context: vmware
+:vmware:
+include::common/modules/con_configuring-virt-who-for-provider.adoc[leveloffset=+1]
+
+include::common/modules/proc_creating-a-virt-who-configuration.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-projectserver.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-hypervisor-smartproxy.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-el.adoc[leveloffset=+2]
+:!vmware:
+:context: nutanix
+:nutanix:
+include::common/modules/con_configuring-virt-who-for-provider.adoc[leveloffset=+1]
+
+include::common/modules/proc_creating-a-virt-who-configuration.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-projectserver.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-hypervisor-smartproxy.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-el.adoc[leveloffset=+2]
+:!nutanix:
+:context: hyperv
+:hyperv:
+include::common/modules/con_configuring-virt-who-for-provider.adoc[leveloffset=+1]
+
+include::common/modules/proc_creating-a-virt-who-configuration.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-projectserver.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-hypervisor-smartproxy.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-el.adoc[leveloffset=+2]
+:!hyperv:
+:context: openstack
+:openstack:
+include::common/modules/con_configuring-virt-who-for-provider.adoc[leveloffset=+1]
+
+include::common/modules/proc_creating-a-virt-who-configuration.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-hypervisor-smartproxy.adoc[leveloffset=+2]
+:!openstack:
+:context: kubevirt
+:kubevirt:
+include::common/modules/con_configuring-virt-who-for-provider.adoc[leveloffset=+1]
+
+include::common/modules/proc_creating-a-virt-who-configuration.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-hypervisor-smartproxy.adoc[leveloffset=+2]
+:!kubevirt:
```

Comment:
Here you are unsetting the whole attribute, which is why attempting to use it later results in an error.